### DISCUSSION TITLE
Bun not resolving escaped single quote?

### DISCUSSION THREAD

**Discussion by dwicks99** [uptrace/bun#853] [Q&A]
When I submit a string with an escaped single quote in it like "hello '' hello" I am expecting Bun to insert it as "hello ' hello" but it just inserts the two single quotes as is and does not parse them into one? I have tried adding some other escape sequences and Bun does not seem to respect them. Am I looking at this wrong?

**Comment by vmihailenco** [comment]
Bun automatically escapes strings. No need to escape strings manually.
